perfscale_regression_ci/scripts/scalability/replace_nodes/replacenodes.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import datetime
+from utils.ocp_utils import run, get_machine_replicas, scale_machine_replicas, wait_for_node_deletion, wait_for_node_creation, cluster_health_check, delete_machineset
```

Comment:
here you could just do **import utils.ocp_utils as ocp_utils** and then call the function by ocp_utils.<function_name>
It would look a little cleaner and don't have to write out all the function names